datalad/interface/tests/test_run.py

```diff
@@ -70,16 +88,15 @@ def test_invalid_call(path):
         # no dataset, no luck
         assert_raises(NoDatasetArgumentFound, run, 'doesntmatter')
         # dirty dataset
-        ds = Dataset(path).create()
+        ds = Dataset(path).rev_create()
         create_tree(ds.path, {'this': 'dirty'})
         assert_status('impossible', run('doesntmatter', on_failure='ignore'))
 
 
-@known_failure_windows
 @with_tempfile(mkdir=True)
 @with_tempfile(mkdir=True)
 def test_basics(path, nodspath):
-    ds = Dataset(path).create()
+    ds = Dataset(path).rev_create()
```

Comment:
Without any arguments, is there a difference? If not, I wonder if it wouldn't be better to keep the ultimate target having plain `create` in the tests, so they won't even need to be adjusted when the switch over comes?

Reply:
The tests coming in from -revolution already use `rev_create`, so I would prefer to just use it consistently throughout test_run.py, as stated in eeed2a8.

Reply:
Got it, thanks!